### PR TITLE
Lc problem 1 two sum

### PR DESCRIPTION
**Pull request by CHAriina**

### CODE REVIEW

**Comment by CHAriina** [comment]
#1 The solution for two sum problems

File: ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            var solution = new TwoSum1.Solution();
+            int[] nums = new int[] { 2, 7, 11, 15 };
+            int target = 9;
+
+            // Act
+            int[] result = solution.TwoSum(nums, target);
+
+            // Assert
+            Assert.Equal(new int[] { 0, 1 }, result);
+
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs:21]: remove empty line

File: ScnSoft-Education/LeetCodeTraining/TwoSum1/TwoSumSolution.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            {
+                for (int j = i + 1; j < nums.Length; j++)
+                {
+                    if (nums[i] + nums[j] == target)
+                    {
+                        return new int[] { i, j };
+                    }
+                }
+            }
+            return null;
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/TwoSum1/TwoSumSolution.cs:19]: add empty line before 'return'

File: ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs (hunk 1 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs:21]: removed

File: ScnSoft-Education/LeetCodeTraining/TwoSum1/TwoSumSolution.cs (hunk 2 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining/TwoSum1/TwoSumSolution.cs:19]: added

**Pull request merged**